samples/gltf_viewer.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <memory>
```

Comment:
We have an ordering convention for includes (unfortunately not followed everywhere yet) which is that lower layers come last:

 https://github.com/google/filament/blob/master/CODE_STYLE.md#headers

For example, if you need to include header A and header B, and you can imagine header A possibly including header B, then B should come after A.